2021/classes/submarine.py

```diff
@@ -0,0 +1,22 @@
+from turtle import numinput
+
+
+class Submarine:
+    def __init__(self, readings):
+        self.readings = readings
+        self.simple_increases = 0
+        self.deep_increases = 0
+
+    @staticmethod
+    def sonar_sweep(readings):
+        return len([True for r1, r2 in zip(readings, readings[1:]) if r2 > r1])
```

Comment:
I get that this method _can_ be static, but in general I believe you should have a very good reason to make static methods, as they aren't tied to anything other than a class definition, which can get problematic if we're working in a system that has some level of dependency injection.

By making this static, we're advertising that our `Submarine` has a public static method, when really we don't want anyone to be using it.

I'd likely rather see this:
```suggestion
    def _sonar_sweep(self, readings):
        return len([True for r1, r2 in zip(readings, readings[1:]) if r2 > r1])
```

Note that `_` prefix, it's to denote that the method is private, as is convention in Python